web_accounts_list.json

```diff
@@ -4711,6 +4711,28 @@
            "known_accounts" : ["amiw", "turbolove"],
            "category" : "social",
            "valid" : true
+          },
+          {
+           "name" : "thesims.com",
+           "check_uri" : "https://forums.thesims.com/en_US/profile/{account}",
+           "account_existence_code" : "200",
+           "account_existence_string" : "The Sims Forums",
+           "account_missing_string" : "User Not Found",
+           "account_missing_code" : "404",
+           "known_accounts" : ["babykittyjade", "simgurumorgan"],
+           "category" : "gaming",
+           "valid" : true
+          },
+          {
+           "name" : "ubisoft.com",
+           "check_uri" : "https://discussions.ubisoft.com/user/{account}",
+           "account_existence_code" : "200",
+           "account_existence_string" : "Ubisoft Discussion Forums",
+           "account_missing_string" : "Not Found",

```

Comment:
Let's be more precise here as you could have profiles with the words `Not Found` in them. Consider changing to `<title>Not Found | Ubisoft`

Reply:
Will do.